CMakeLists.txt

```diff
@@ -164,10 +164,30 @@ add_subdirectory( core )
 add_subdirectory( device/common )
 if( TRACCC_BUILD_CUDA )
    add_subdirectory( device/cuda )
+else()
+   if( TRACCC_BUILD_KOKKOS )
+      add_subdirectory( device/cuda )
+   endif()
 endif()
```

Comment:
Let's simplify this a bit. The following would look a bit better:

```cmake
if( TRACCC_BUILD_CUDA OR TRACCC_BUILD_KOKKOS )
   add_subdirectory( device/cuda )
endif()
```

Although I do not like that there would be an inter-dependency between these libraries. :frowning: And as much as I can see, this inter-dependence is not even there anymore anyway.